.vscode/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build",
+            "command": "dotnet",
+            "type": "process",
+            "args": [
+                "build",
+                "${workspaceFolder}/tests/AnonymousUserTests/AnonymousUserTests.csproj",
```

Comment:
Should we target the whole solution instead?

Reply:
Not needed here, this is for debugging the code only.